docs/pipeline.rst

```diff
@@ -520,6 +520,7 @@ tokenizer_jieba
 
         pipeline:
         - name: "tokenizer_jieba"
+          dictionary_path: "path/to/custom/dictionary/dir"  # or None (which is default value) means don't use custom dictionary
```

Comment:
please add a note that the path needs to be valid when training and loading the model as the dictionary is not persisted as part of the model.

Reply:
Maybe it's better that allow user to choice persist dictionary to model or not? If user load models from cloud-based storage, it will be better that models provide dictionary file, there maybe need several different dictionaries at same time from different models, server-level config don't have such ability.

Reply:
I mean the configuration is still model based and stored with the model. So the path is persisted with the model, which means every model can use a different dictionary, but the path needs to be the same as during training.

How large are these dictionaries?

Reply:
The size of dictionary file varies from few bytes to few megabytes  depends on user's application. In most cases,  the size smaller than 1 KB.

Reply:
yeah in that case I think it is better to copy it to the persisted model

Reply:
Sure, I will update the code very soon.

Reply:
Awesome, thanks a lot for the ongoing effort :rocket: